docs/end_to_end_acknowledgements.md

```diff
@@ -0,0 +1,63 @@
+
+## End-to-end Acknowledgements
```

Comment:
I think we are settling on no "e", so this should be `acknowledgments`.

Reply:
Yeah. That's only for external facing (config) word. I am being little flexible on general usage of the word.